server/app/services/languageProcessor/DecisionTree.java

```diff
@@ -18,71 +18,64 @@
 		root.left.right.left.left = new Node("getTicketAssignee");
 		root.left.right.left.right = new Node("reporter_of_ticket");
 	}
-	
+
 	public void insertRoot(Node node){
 		if (root == null)
 			root = node;
```

Comment:
Please fix the formatting of this file and always include `{ }` for `if else` block.